rust/catalyst-types/src/uuid/mod.rs

```diff
@@ -4,8 +4,10 @@ mod uuid_v4;
 mod uuid_v7;
 
 use minicbor::data::Tag;
-pub use uuid_v4::UuidV4 as V4;
-pub use uuid_v7::UuidV7 as V7;
+#[allow(clippy::module_name_repetitions)]
+pub use uuid_v4::UuidV4;
+#[allow(clippy::module_name_repetitions)]
```

Comment:
Instead of using the `clippy::module_name_repetitions` attribute it is possible to rename inner modules. These module are private anyway, so it wouldn't break any existing code. I have used the attribute to minimize changes, but I can do it the other way.

Reply:
I dont oppose to rename the inner modules as well